man/fi_av.3.md

```diff
@@ -359,9 +359,10 @@ The total number of inserted addresses will be nodecnt x svccnt.
 ## fi_av_remove
 
 fi_av_remove removes a set of addresses from an address vector.  All
-resources associated with the indicated addresses are released, and no
-future references to either the mapped address (in the case of
-FI_AV_MAP) or index (FI_AV_TABLE) are allowed.
+resources associated with the indicated addresses are released.
+The removed address - either the mapped address (in the case of FI_AV_MAP)
+or index (FI_AV_TABLE) is invalid until it is returned again by a
```

Comment:
I'm sorry to be super picky...

If you start a phrase with "-", then you need to finish it with a "-", too:

> The removed address -- either the mapped address (in the case of FI_AV_MAP) or index (FI_AV_TABLE) -- is invalid until it is returned again by a ...